Description: Add URL to several files
 In README and FAQ, add the http://saftsack.fs.uni-bayreuth.de/~latex2ht/ URL
Author: Roland Stigge <stigge@antcom.de>

--- latex2html-2008-debian1.orig/README
+++ latex2html-2008-debian1/README
@@ -170,6 +170,8 @@ Support and More Information
 
 Announcements, discussion archives, bug reporting forms and
 more are kept at the LaTeX2HTML home at 
+http://saftsack.fs.uni-bayreuth.de/~latex2ht/
+or
 http://cbl.leeds.ac.uk/nikos/tex2html/doc/latex2html/latex2html.html.
 
 *** this site has not been updated since November 1996.

--- latex2html-2008-debian1.orig/FAQ
+++ latex2html-2008-debian1/FAQ
@@ -170,9 +170,9 @@ change_end 97.1
    
    
    Alternatively, a compressed tar  file of the source and related files
-   may be obtained via anonymous ftp  to
-   http://www-dsed.llnl.gov/files/programs/unix/latex2html/sources/latex2
-   html-98.1.tar.gz.
+   may be obtained from
+   http://saftsack.fs.uni-bayreuth.de/~latex2ht/
+
    
    Two other ftp -sites are
    http://ctan.tug.org/ctan/tex-archive/support/latex2html and
@@ -191,7 +191,8 @@ change_end 97.1
    
    
    Updates and patches are posted on the LATEX2HTML server at
-   http://www-dsed.llnl.gov/files/programs/unix/latex2html/.
+   http://saftsack.fs.uni-bayreuth.de/~latex2ht/
+
 change_end 98.1
    
    
@@ -630,8 +631,9 @@ change_end 98.1
      * To join the community of LATEX2HTML users:
        More information on a mailing list, discussion archives, bug
        reporting forms and more is available at
-       http://cbl.leeds.ac.uk/nikos/tex2html/doc/latex2html/latex2html.ht
-       ml
+       http://saftsack.fs.uni-bayreuth.de/~latex2ht/
+       or
+       http://cbl.leeds.ac.uk/nikos/tex2html/doc/latex2html/latex2html.html
        
    2.2 Getting Support and More Information
 
